be/src/util/ratelimit.h

```diff
@@ -0,0 +1,17 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+#pragma once
+
+#include <atomic>
+
+#include "util/time.h"
+
+#define RATE_LIMIT(func, interval_ms)          \
```

Comment:
glog has it's own LOG_EVERY_N macro to reduce log, can have a try.

Reply:
use RATE_LIMIT because of two reason:
1. LOG_EVERY_N limit printing log by count, which is difficult to use than limit printing log by time interval. 
2. RATE_LIMIT can't only be used in printing log, it can be useful to do any function call limit later.